core/schema/secret.go

```diff
@@ -72,7 +80,7 @@ func (s *secretSchema) setSecret(ctx context.Context, parent *core.Query, args s
 	// NB: to avoid putting the plaintext value in the graph, return a freshly
 	// minted Object that just gets the secret by name
 	if err := s.srv.Select(ctx, s.srv.Root(), &i, dagql.Selector{
-		Field: "secret",
+		Field: "loadSecretFromName",
```

Comment:
nit: could just update `setSecret` to call `secret` now with `plaintext-insecure://` or whatever URI now right? I think that might be a bit easier to follow and de-dupe a little code

Reply:
Downside of `plaintext-insecure://` (which is not implemented) is it's going to end up in cache, right?

Which is fine for new code, but less fine IMHO to "downgrade" `setSecret` from somewhat secure to insecure.

Reply:
It *should* be fine because nothing is cached on-disk and there's various protections around handling sensitive arg values so they don't get displayed or sent as telemetry. But I'm fine leaving this as-is to avoid needing to risk anything and needing a bunch of extra testing to validate it.